saleor/graphql/order/types.py

```diff
@@ -340,6 +342,10 @@ class OrderEvent(ModelObjectType[models.OrderEvent]):
     related_order = graphene.Field(
         lambda: Order, description="The order which is related to this order."
     )
+    related = graphene.Field(
+        lambda: OrderEvent,
+        description="The order event which is related to this event." + ADDED_IN_314,
```

Comment:
Shouldn't we also mark this field as a PREVIEW_FEATURE? The mutation that can set this field is in preview, so I think would be also consider adding PREVIEW_FEAUtRE here also